Doc/library/timeit.rst

```diff
@@ -233,6 +243,12 @@ Where the following options are understood:
 
    .. versionadded:: 3.5
 
+.. cmdoption:: -m, --max_time_taken=M
+
+    calls :meth:`.timeit` repeatedly so that the total time >= *max_time_taken* second
+
+   .. versionadded:: 3.7
```

Comment:
3.8 as 3.7 is already feature freeze

Reply:
Hello and thanks for the review. 
I wrote 3.7 because it was the lowest version setted in https://bugs.python.org/issue36461 by @csabella 
As you said correctly (like also PEP-537 states) 3.7 is now frozen. I already made the changes.